mirdata/datasets/saraga_carnatic.py

```diff
@@ -621,17 +475,13 @@ def load_phrases(phrases_path):
     events = []
     with open(phrases_path, 'r') as reader:
         for line in reader.readlines():
+            start_times.append(float(line.split('\t')[0]))
+            end_times.append(
+                float(line.split('\t')[0]) + float(line.split('\t')[2])
+            )
             if len(line.split('\t')) == 4:
-                start_times.append(float(line.split('\t')[0]))
-                end_times.append(
-                    float(line.split('\t')[0]) + float(line.split('\t')[2])
-                )
                 events.append(str(line.split('\t')[3].split('\n')[0]))
-            if len(line.split('\t')) == 3:
-                start_times.append(float(line.split('\t')[0]))
-                end_times.append(
-                    float(line.split('\t')[0]) + float(line.split('\t')[2])
-                )
+            else:
                 events.append('No information')
```

Comment:
Maybe an empty event should be simply " "? Just wondering if adding a str there is not misleading

Reply:
Got it! Yeah I think that's better than adding uneeded confusing strings.